src/SearchPanel.jsx

```diff
@@ -8,24 +8,24 @@ import { shortIsoFormat } from './utils'
 
 export class SearchPanel extends React.Component {
   render() {
-    if (this.props.lastUpdated) {
+    if ( this.props.lastUpdated ) {
```

Comment:
You don't have to rewrite the whole HTML block, the code could be simplified to:

```Typescript
let e =  null
if ( this.props.lastUpdated ) {
  e = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
}

...

<h2>Search complaint data { e }</h2>
```